pkg/apis/extensions/v1beta1/zz_generated.conversion.go

```diff
@@ -941,6 +941,18 @@ func Convert_autoscaling_HorizontalPodAutoscalerList_To_v1beta1_HorizontalPodAut
 	return autoConvert_autoscaling_HorizontalPodAutoscalerList_To_v1beta1_HorizontalPodAutoscalerList(in, out, s)
 }
 
+func autoConvert_v1beta1_HorizontalPodAutoscalerSpec_To_autoscaling_HorizontalPodAutoscalerSpec(in *HorizontalPodAutoscalerSpec, out *autoscaling.HorizontalPodAutoscalerSpec, s conversion.Scope) error {
+	out.MinReplicas = in.MinReplicas
+	out.MaxReplicas = in.MaxReplicas
```

Comment:
It seems like a bug for me - basically both types contain two other fields:
- ScaleRef (ScaleTargetRef)
- CPUUtilization (TargetCPUUtilizationPercentage)

We should probably print some warning for those?